testing_log/02-17-20.txt

```diff
@@ -0,0 +1,236 @@
+First test run of test_driver.py:
```

Comment:
What is this txt file? Where did it come from?

Reply:
I just thought it would be good to write down/record everything I did during the testing of the code and make a note of the changes I made and why.